src/app/profile/settings/notifications/notifications.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="container-fluid container-cards-pf">
+  <div class="row-cards-pf">
+    <div class="col-xs-10 col-sm-6 col-md-6">
+      <div class="card-pf">
+        <div class="card-pf-heading">
+          <div class="row notification-option-content">
+            <div class="col-xs-6 col-sm-6 col-md-6">
+              <h2 class="card-pf-title">Delivery Methods</h2>
+            </div>
+          </div>
+        </div>
+        <div class="card-pf-body">
+          <div class="row github-notification-option-info">
+            <div class="col-xs-1 col-sm-1 col-md-1">
+              <input type="checkbox" [disabled]="true" [checked]="true">
+            </div>
+            <div class="notification-option-text-box col-xs-10 col-sm-10 col-md-10">
+              <b>System Widget Notifications</b>
+              <br>
+              <p>These will only be presented in the notification drawer, represented by the [] icon.</p>
+            </div>
+          </div>
+          <div class="row github-notification-option-info">
+            <div class="col-xs-1 col-sm-1 col-md-1">
+                <input type="checkbox" [disabled]="true" [checked]="true">
+              </div>
+            <div class="notification-option-text-box col-xs-10 col-sm-10 col-md-10">
+              <b>Email Notifications</b>
+              <br>
+              <p>These will only be sent to the primary email listed in your
+                <a [routerLink]="'/' + [loggedInUserName]">user profile</a>.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-xs-2 col-sm-6 col-md-6"></div>
+  </div>
+</div>
+<div class="container-fluid container-cards-pf">
+  <div class="row-cards-pf">
+    <div class="col-xs-10 col-sm-6 col-md-6">
+      <div class="card-pf">
+        <div class="card-pf-heading">
+          <div class="row notification-option-content">
+            <div class="col-xs-6 col-sm-6 col-md-6">
+              <h2 class="card-pf-title">Global Notification Triggers</h2>
+            </div>
+          </div>
+        </div>
+        <div class="card-pf-body">
+          <div class="row github-notification-option-info">
+            <div class="col-xs-1 col-sm-1 col-md-1">
+                <input type="checkbox" [disabled]="true" [checked]="true">
+              </div>
+            <div class="notification-option-text-box col-xs-11 col-sm-11 col-md-11">
+              <b>Assigned to an issue</b>
+              <br>
```

Comment:
Preferred method is to use a margin on the element above - "Assigned to an issue".